### DISCUSSION TITLE
Landing Page MapLibre Native

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre.github.io#364] [Ideas]
I think it would be useful to have a landing page for MapLibre Native on the website. The current one line summary and screenshots on the website don't do the project and its ecosystem justice.

Maybe some quotes from users. Where the focus should be on MapLibre Native (it's not some sponsorship thing). And maybe pictures from MapLibre Native used in the real world.

It should prominently feature the platforms MapLibre Native has bindings for. Not just Android and iOS, but also Qt, React Native, Node.js, Kotlin Multiplatform, Flutter. We should also put the logos of Metal, OpenGL and Metal.

We should not just focus it on (potential) users but also use it to draw in (potential) contributors. For example by making it clear that it is a polyglot project, show a simplified drawing of the architecture and a link to the architecture documentation to learn more.

Let's gather some more ideas.

**Comment by HarelM** [comment]
I think a landing page should be short and precise. 
We can think of a "read more..." kind of button to allow expanding the short description to more information, but I wouldn't want to change the current design dramatically, I think it's good. 

It might be a good idea to dedicate a page in this site for each core project under projects or products and place this extra information there, or in the landing page of the "main" docs of each project.

That's my 2 cents.

I would also argue that Martin doesn't need to be on the main page or we should be adding more projects like Maputnik (and maybe others).

**Reply by louwers** [reply]
Agreed that the general landing page is good, and I don't want to change it too much. Although I think I would prefer just one button to go to the project overview page I am suggesting. There's already many buttons for MapLibre Native, it's hard to find the right one and they don't even link to half of the relevant locations.

W.r.t Martin. It's the second most starred project in the MapLibre GitHub org, it's used productively in the wild and it shows that MapLibre is more than about just the renderers and their ecosystems. So I think it's good to have it on the landing page. Although we could make MapLibre GL JS and MapLibre Native visually distinct from other projects.

**Reply by HarelM** [reply]
Not sure stars are the right criteria, and I think we should better define who appears in the main page and who doesn't.
![image](https://github.com/user-attachments/assets/7c97169d-4546-47f2-be50-28d141732024)

In any case, a project page is a good solution to better explain the links instead of these not-so-intuitive buttons ,and allow some overview and in general a more detailed "landing page" for each product/project/library.

**Comment by birkskyum** [comment]
@louwers , fyi , the maplibre.org is using astro now. It's file based routing - i've added a placeholder file to demonstrate how a project page route can be added for native:

https://github.com/maplibre/maplibre.github.io/blob/main/src/pages/projects/_maplibre-native.astro

The underscore in front disables the route, so it's not live. Removing it will make a /projects/maplibre-native route with the content shown in the video below:



 
https://github.com/user-attachments/assets/73ebada9-141f-4f57-b452-9a782e163b6a

**Reply by louwers** [reply]
Oh wow. Snappy.
